db.json

```diff
@@ -43,7 +43,7 @@
       "genre": "Drama, Romance",
       "rating": 8.4,
       "year": "1994",
-      "poster": "https://images.app.goo.gl/6R1zNbYMpsa2RQZ67"
+      "poster": "https://m.media-amazon.com/images/M/MV5BMTY4YjFhMzUtMWIxMC00NDI2LTgxOGYtMDM1ZGQ1ZDUxZTE0XkEyXkFqcGdeQXVyMTUzOTcyODA5._V1_.jpg"
```

Comment:
You added a poster for a different movie. Please check :P

Reply:
Okay, let me fix it.